### PR TITLE
Ankit | Test | Prod - Confirmation pop up should come on delete session option 

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cw54agw

### CODE REVIEW

File: apps/web-giddh/src/app/user-details/user-details.component.ts [hunk 1]
```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
     }
 
     public clearAllSession() {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/user-details/user-details.component.ts]: Add function return Type and also write documentation

File: apps/web-giddh/src/app/user-details/user-details.component.ts [hunk 2]
```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
...
+            color: 'primary'
+        },
+        {
+            text: this.commonLocaleData?.app_no
+        }];
+        const headerText: string =  this.commonLocaleData?.app_confirmation;
+        const headerCssClass: string = 'd-inline-block mr-1';
+        const messageCssClass: string = 'mr-b1';
+        const messageText: String = this.localeData?.session?.delete_all_session;
+        let configuration = {buttons, headerText, headerCssClass, messageCssClass, messageText}
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/user-details/user-details.component.ts]:
```suggestion
        const configuration = { buttons, headerText, headerCssClass, messageCssClass, messageText };
```

File: apps/web-giddh/src/app/user-details/user-details.component.ts [hunk 3]
```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
...
+        
+        let dialogRef = this.dialog.open(NewConfirmationModalComponent, {
+            width: '630px',
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/user-details/user-details.component.ts]:
```suggestion
           panelClass: ['mat-dialog-md']
```

File: apps/web-giddh/src/app/user-details/user-details.component.ts [hunk 4]
```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
...
+            data: {
+                configuration: configuration
+            }
+        });
+
+        dialogRef.afterClosed().pipe().subscribe(response => {
+            if (response === 'Yes') {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/user-details/user-details.component.ts]:
```suggestion
            if (response === this.commonLocaleData?.app_yes) {
```

File: apps/web-giddh/src/app/user-details/user-details.component.ts [hunk 5]
```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
...
+                this.store.dispatch(this.sessionAction.deleteAllSession());
+            } else {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/user-details/user-details.component.ts]: remove this else if not in use

File: apps/web-giddh/src/assets/locale/user-details/mr.json [hunk 6]
```diff
@@ -48,7 +48,9 @@
 		"signin_time": "साइन इन वेळ",
 		"duration": "कालावधी (डीडी / एचएच / एमएम / एसएस)",
 		"browser_agent": "ब्राउझर एजंट",
-		"current": "करंट"
+		"current": "करंट",
+		"delete_all_session": "तुम्हाला खात्री आहे की तुम्ही सर्व सत्रे हटवू इच्छिता, सर्व"
+
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/locale/user-details/mr.json]: remove extra line

File: apps/web-giddh/src/assets/locale/user-details/en.json [hunk 7]
```diff
@@ -48,7 +48,8 @@
         "signin_time": "Signin Time",
         "duration": "Duration (DD/HH/MM/SS)",
         "browser_agent": "Browser Agent",
-        "current": "Current"
+        "current": "Current",
+        "delete_all_session": "Are you sure you want to delete all the sessions,all the"
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/assets/locale/user-details/en.json]: Check and confirm message from Kriti and translate as well.

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/user-details/user-details.component.ts (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/user-details/user-details.component.ts]: DONE

File: apps/web-giddh/src/app/user-details/user-details.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/user-details/user-details.component.ts]: Done

File: apps/web-giddh/src/app/user-details/user-details.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/user-details/user-details.component.ts]: Done

File: apps/web-giddh/src/app/user-details/user-details.component.ts (hunk 4 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/user-details/user-details.component.ts]: Done

File: apps/web-giddh/src/assets/locale/user-details/mr.json (hunk 6 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/assets/locale/user-details/mr.json]: Done

File: apps/web-giddh/src/assets/locale/user-details/en.json (hunk 7 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/assets/locale/user-details/en.json]: DONE

**Pull request merged**